src/snowflake/snowpark/mock/_connection.py

```diff
@@ -229,6 +236,47 @@ def __init__(self, options: Optional[Dict[str, Any]] = None) -> None:
         self._active_schema = self._options.get(
             "schema", snowflake.snowpark.mock._constants.CURRENT_SCHEMA
         )
+        self._connection_uuid = str(uuid.uuid4())
```

Comment:
Shall we create a TODO for to use the `.snowsql` approach so we can track telemetry per installation instead?